test/unit/foreman/access_permissions_test.rb

```diff
@@ -10,59 +11,8 @@
 #
 # In particular, it's important that actions for AJAX requests are added to
 # an appropriate permission so views using those requests function.
-class AccessPermissionsTest < ActiveSupport::TestCase
-  MAY_SKIP_REQUIRE_LOGIN = [
-    "users/login", "users/logout", "users/extlogin", "users/extlogout", "home/status", "notices/destroy",
+class AccessPermissionTest < ActiveSupport::TestCase
```

Comment:
Why rename this? It should still be plural I think to match the class and filename.